src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,148 @@
+package duke;
+
+import javafx.application.Application;
+import javafx.geometry.Pos;
+import javafx.scene.Scene;
+import javafx.scene.control.Label;
+import javafx.stage.Stage;
+import java.util.ArrayList;
+import javafx.scene.control.Button;
+import javafx.scene.layout.HBox;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.input.KeyCode;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.VBox;
+import javafx.scene.layout.Region;
+
+/**
+ * Duke is a task management chatbot GUI that allows users to manage their tasks
+ * including todos, deadlines, and events.
+ */
+public class Duke extends Application {
+
+    private ScrollPane scrollPane;
+    private VBox dialogContainer;
+    private TextField userInput;
+    private Button sendButton;
+    private Scene scene;
+    private String outputText;
+
+    @Override
+    public void start(Stage stage) {
+        Storage storage = new Storage("src/main/java/duke/data/duke.txt");
+        TaskList taskList = new TaskList();
+
+        Ui.getGreeting();
+
+        ArrayList<Task> loadedTasks = storage.loadTasks();
+        taskList.setTasks(loadedTasks);
+
+        scrollPane = new ScrollPane();
+        dialogContainer = new VBox();
+        scrollPane.setContent(dialogContainer);
+
+        userInput = new TextField();
+        sendButton = new Button("Send");
+
+        AnchorPane mainLayout = new AnchorPane();
```

Comment:
Is there a better place to put this logic? 

Perhaps another class such as MainWindow?